lte/gateway/python/magma/enodebd/tr069/spyne_mods.py

```diff
@@ -140,6 +140,15 @@ def get_call_handles(self, ctx):
         return super(Tr069Soap11, self).get_call_handles(ctx)
 
     def serialize(self, ctx, message):
+        # Workaround for issue https://github.com/magma/magma/issues/7869
+        # Updates to ctx.descriptor.out_message.Attributes.sub_name are taking
+        # effect on the descriptor. But when puled from _attrcache dictionary,
+        # it still has a stale value.
+        # Force repopulation of dictionary by deleting entry
+        # TODO Remove this code once we have a better fix
+        if (self._attrcache.__contains__(ctx.descriptor.out_message)):
+            self._attrcache.__delitem__(ctx.descriptor.out_message)
+
```

Comment:
There is a Baicells test where we simulate inputs if you want to add a test there, you can get rid of this. But optional.

Reply:
State machine tests do not cover TR69 Soap11 functions. And TR69 Soap11 tests are simple single message tests.
This requires a new test that combines a bit of both